components/Behaviors/src/MultiTarget.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+    <PropertyGroup>
+        <!--
+          MultiTarget is a custom property that indicates which target a project is designed to be built for / run on.
+          Used to create project references, generate solution files, enable/disable TargetFrameworks, and build nuget packages.
+        -->
+        <MultiTarget>uwp;wasdk;wpf;wasm;linuxgtk;macos;ios;android;</MultiTarget>
```

Comment:
Since this is only good for WinUI 3, remove `uwp` here. @Arlodotexe that should exclude it from making an entire UWP package for Uno too, eh? Then we wouldn't need the `#if WINAPPSDK` conditional, right?

Reply:
Yes this feature only works for wasdk. But sorry that I'm not familiar with `tooling` project, so don't know how exactly i should do. I hope you can help me fix this❤️thanks

Reply:
@Arlodotexe maybe something we can lend a quick hand with this after the long weekend here?